src/pymovements/utils/downloads.py

```diff
@@ -27,7 +27,7 @@
 
 from tqdm.auto import tqdm
 
-from pymovements.utils.archives import extract_archive
+from pymovements.dataset._utils._archives import extract_archive
```

Comment:
in the previous PR you had this as `public_module` as `_public_module`

why not here?

Reply:
Actually I liked the way you did it in #914 with `dataset/_utils/_yaml.py` better, but #976 was already merged by then.